metadata.json

```diff
@@ -87,6 +87,10 @@
         {
             "name": "puppetlabs/stdlib",
             "version_requirement": ">= 4.2.2 <5.0.0"
+        },
+        {
+            "name": "puppetlabs/ntp",
```

Comment:
Let's discuss this in https://github.com/puppetlabs/puppetlabs-puppetdb/pull/263